src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/components/Results.tsx

```diff
@@ -100,11 +100,9 @@ export const Results: React.FC<Props> = ({
               departureTime={tripPattern.expectedStartTime}
               previousDepartureTime={tripPatterns[i - 1]?.expectedStartTime}
             />
-            <ResultItem
+            <MemoizedResultItem
               tripPattern={tripPattern}
-              onDetailsPressed={() => {
-                onDetailsPressed(tripPattern, i);
-              }}
+              onDetailsPressed={() => onDetailsPressed(tripPattern, i)}
```

Comment:
When `React.memo` is used you can't inline function creation like done for `onDetailsPressed` here. This will recreate the function for every render, thus props to `MemoizedResultItem` is changed for every render, and as such `React.memo` has no effect.